### PR TITLE
Docs: What’s new & Upgrade guide v11.2

### PR DESCRIPTION
**Pull request by imatwawana**
**CONTRIBUTORS, PLEASE DO NOT ADD DIRECTLY TO THIS PR**

- **For What's new**: Create entries using the website CMS
- **For Upgrade guide**: Create a branch and make changes there for review prior to merging

****************************************************************************************
**REVIEWERS**: 

- Only make placement changes to entries.
- Only make copy edits directly except to the introductory text. 
- To update text of entries, please update them in the What's new in Cloud entry and notify me.

### CODE REVIEW

File: docs/sources/upgrade-guide/upgrade-v11.2/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Upgrade to Grafana v11.2
+
+{{< docs/shared lookup="upgrade/intro.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+{{< docs/shared lookup="back-up/back-up-grafana.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}
+
+{{< docs/shared lookup="upgrade/upgrade-common-tasks.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+## Technical notes
```
Comment (jdbaldry) [inline docs/sources/upgrade-guide/upgrade-v11.2/index.md:22]: Is there content expected or should we remove this or is it OK to leave an empty heading?

**Review by jdbaldry — APPROVED** [review]
LGTM

File: docs/sources/upgrade-guide/upgrade-v11.2/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/upgrade-guide/upgrade-v11.2/index.md:22]: we typically leave it tbh

File: docs/sources/whatsnew/whats-new-in-v11-2.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+title: What's new in Grafana v11.2
+weight: -44
+---
+
+<!-- vale GoogleWe = NO -->
+<!-- vale We = NO -->
+
+# What’s new in Grafana v11.2
+
+Welcome to Grafana 11.2! We've made a number of improvements in this release including, most excitingly, the introduction of the Grafana Cloud Migration Assistant in public preview. We've also added several new transformations, added a centralized page for viewing your alert history, and made updates to improve and simplify authentication for your Grafana instance.
```
Comment (mjseaman) [inline docs/sources/whatsnew/whats-new-in-v11-2.md:23]:
```suggestion
Welcome to Grafana 11.2! We've made a number of improvements in this release, including a Grafana Cloud Migration Assistant in public preview, several new transformations, and a centralized page for viewing your alert history. We've also released several new data sources to help you visualize data from Zendesk, Catchpoint, and Yugabyte. Read on to learn more about these and all the new features in v11.2.
```

File: docs/sources/whatsnew/whats-new-in-v11-2.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+**New cumulative and window calculations available in Add field from calculation**
+
+The **Add field from calculation** transformation now supports both cumulative and window calculations. The cumulative function calculates on the current row and all preceding rows. You can calculate the total or the mean of your data up to and including the current row. With the window function you can calculate the mean, standard deviation, or variance on a specified set (window) of your data. The window can either be trailing or centered. With a trailing window the current row will be the last row in the window. With a centered window the window will be centered on the current row.
+
+See [the documentation for more information](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation).
+
+### Improvements in canvas visualizations
```
Comment (mjseaman) [inline docs/sources/whatsnew/whats-new-in-v11-2.md:93]:
```suggestion
### Improvements to the Canvas panel
```

Comment (imatwawana) [reply docs/sources/whatsnew/whats-new-in-v11-2.md:93]: A panel and a visualization are two different things in our UI and we want to help our customers get used to that. Calling them visualizations and lower casing these is a big shift we've made over the past year in the technical documentation that we want reflected here.

**Pull request merged**